cluster-scope/base/objectbucket.io/objectbucketclaims/acm-metrics/objectbucketclaim.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: acm-metrics
+  namespace: open-cluster-management-observability
+spec:
+  generateBucketName: acm-metrics
+  storageClassName: openshift-storage.noobaa.io
```

Comment:
This should use the new storageClassName `acm-metrics-bucket-storage` created in the earlier PR.